pkg/terra/config.go

```diff
@@ -18,7 +18,12 @@ var defaultConfigSet = configSet{
 	BlocksUntilTxTimeout:  10,
 	ConfirmPollPeriod:     time.Second,
 	FallbackGasPriceULuna: sdk.MustNewDecFromStr("0.015"),
-	GasLimitMultiplier:    1.5,
+	// This is high since we simulate before signing the transaction.
```

Comment:
hmm, should we also have `DefaultTimeout` configurable?

Reply:
We can def defer this for later...

Reply:
yeah it should be, can be done in a follow up